cmd/collector/main.go

```diff
@@ -248,6 +267,24 @@ func realMain(ctx *cli.Context) error {
 	addAttributes := cfg.AddAttributes
 	liftAttributes := cfg.LiftAttributes
 
+	sort.Slice(cfg.LiftResources, func(i, j int) bool {
+		return cfg.LiftResources[i].Name < cfg.LiftResources[j].Name
+	})
+
+	logsMapping := schema.DefaultLogsMapping
+	var sortedLiftedResources []string
+	for _, v := range cfg.LiftResources {
+		sortedLiftedResources = append(sortedLiftedResources, v.Name)
+		if v.ColumnName != "" {
+			logsMapping = logsMapping.AddStringMapping(v.ColumnName)
+			continue
+		}
+		logsMapping = logsMapping.AddStringMapping(v.Name)
+	}
+
+	// Update the default mapping so pooled csv encoders can use the lifted columns
+	schema.DefaultLogsMapping = logsMapping
+
```

Comment:
Yeah, probably need a variety of validations for several things.  I can take a crack at some of that in a follow up PR since this is already fairly large.